virttest/utils_test/libvirt.py

```
@@ -3543,28 +3543,58 @@ def update_vm_disk_source(
+        match, output = session.read_until_last_line_matches(
+            [prompt_pattern], timeout=10
+        )
+
+        # Check for virsh errors in the captured output
+        if "error:" in output or "failed:" in output:
+            LOG.error("virsh edit command failed with output:\n%s", output)
+            session.close()
+            return False
+
         session.close()
         return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)